### PR TITLE
Sol-fbp-generator: Add support for OPTIONS in FBP declare.

### PR DESCRIPTION
**Pull request by cabelitos**

### CODE REVIEW

**Comment by cabelitos** [comment]
I forgot to add the licenses to the new fbp tests. I already did that locally.

File: src/bin/sol-fbp-generator/main.c [hunk 1]
```diff
@@ -253,92 +282,125 @@ check_suboption(char *option, const struct sol_fbp_meta *meta, const char *fbp_f
...
-        if (!streq(o->data_type, "string") && meta->value.len > 1
-            && meta->value.data[0] == '"' && meta->value.data[meta->value.len - 1] == '"') {
-            unquoted_meta = *meta;
-            unquoted_meta.value.data += 1;
-            unquoted_meta.value.len -= 2;
-            meta = &unquoted_meta;
-        }
+    if (streq(o->data_type, "int") || streq(o->data_type, "float")) {
+        r = true;
+        if (memchr(meta->value.data, ':', meta->value.len))
```
Comment (lpereira) [inline src/bin/sol-fbp-generator/main.c]: Why not create a function `has_explicit_fields()` that does this `memchr()`?

Comment (cabelitos) [inline src/bin/sol-fbp-generator/main.c]: Does it worth it? anyway, I did not implement this code.

**Comment by cabelitos** [comment]
Closing in favor of v2.

**Pull request closed without merging**